### PR TITLE
Priority mapping of Fatal errors to PSR Log CRITICAL

### PR DESCRIPTION
**Pull request by finwe**
Changes mapping of exceptions as INFO, which is insufficient severity.

### CODE REVIEW

**Comment by finwe** [comment]
And - maybe the default log level should not be INFO but something more severe as we have encountered unexpected behaviour.

File: src/Kdyby/Monolog/Diagnostics/MonologAdapter.php [hunk 1]
```diff
@@ -46,6 +51,15 @@ public function __construct(Monolog\Logger $monolog)
 
 		$this->directory = &Debugger::$logDirectory;
 		$this->email = &Debugger::$email;
+
+		$this->priorityMap = array(
```
Comment (fprochazka) [inline src/Kdyby/Monolog/Diagnostics/MonologAdapter.php]: this could be at the property, doesn't it?

Comment (finwe) [inline src/Kdyby/Monolog/Diagnostics/MonologAdapter.php]: Sure. I am used to assign values to properties in the constructor. Shoud I move it then, I suppose?

Comment (fprochazka) [inline src/Kdyby/Monolog/Diagnostics/MonologAdapter.php]: If it were somehow dynamic, it would be better in construtor, but here it's completely static.

**Comment by fprochazka** [comment]
I definitely like the idea, I just have to think more about the levels that should be used.

**Comment by finwe** [comment]
Could the levels be done separately later, so this could be merged, please? Or are there some concerns with this PR in particular?

**Comment by fprochazka** [comment]
Thank you!

**Comment by finwe** [comment]
Could you tag this, pls? It would also solve the ```Multiple services of type Tracy\ILogger found: tracy.logger, monolog.adapter``` issue with Nette 2.3 when using 1.0.2. Thanks.

**Comment by fprochazka** [comment]
I haven't finished solving compatibility with 2.3. And I'm not sure I'll be able to do it this week.

**Comment by finwe** [comment]
ACK

**Comment by fprochazka** [comment]
But I don't think it's a problem to just run on master for a while, is it? :)

**Comment by finwe** [comment]
Sure, run it on master for a while and suddenly it is broken. But what can I do.

**Comment by fprochazka** [comment]
You can lock the commit, because I don't forcepush to master

```
"kdyby/monolog": "dev-master#hash as 1.0.99"
```

**Comment by finwe** [comment]
Sure, thx.

**Pull request merged**